src/picongpu/submit/hypnos/picongpu.profile.example

```diff
@@ -8,17 +9,17 @@ then
 
         # Core Dependencies
         module load gcc/4.6.2
-        module load cmake/2.8.10
+        module load cmake/3.0.1
         module load openmpi/1.6.3
         module load boost/1.54.0
         module load cuda/6.5
 
         # Plugins (optional)
         module load pngwriter/0.5.4
-        module load hdf5-parallel/1.8.11 libsplash/1.2.2
+        module load hdf5-parallel/1.8.14 libsplash/1.2.3
```

Comment:
*blocked for now:* seems to be compiled against `1.8.11` instead, but admin and module `prereq` say different.